adi/ad9361.py

```diff
@@ -131,8 +131,68 @@ def sample_rate(self):
         return self._get_iio_attr("voltage0", "sampling_frequency", False)
 
     @sample_rate.setter
-    def sample_rate(self, value):
-        self._set_iio_attr("voltage0", "sampling_frequency", False, value)
+    def sample_rate(self, rate):
+        if (rate < 521e3):
+            raise SystemExit("Error: Does not currently support sample rates below 521e3")
```

Comment:
SystemExit might be a tad too severe since it exits the interpreter. Standard exception should be enough